Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -744,8 +743,6 @@ - (void)displayAnyUnseenUpgradeExperience
         ExperienceUpgradesPageViewController *experienceUpgradeViewController =
             [[ExperienceUpgradesPageViewController alloc] initWithExperienceUpgrades:unseenUpgrades];
         [self presentViewController:experienceUpgradeViewController animated:YES completion:nil];
-    } else if (!self.hasEverAppeared && [ProfileViewController shouldDisplayProfileViewOnLaunch]) {
```

Comment:
At this point this just seems like a liability. Happy to put it back if you think it's useful.